### PR TITLE
creates xcode project

### PR DESCRIPTION
**Pull request by DanWebster**

### CODE REVIEW

**Comment by DanWebster** [comment]
@syoung-smallwisdom : unofficial review request if you please
And can I have write access on this repo when you get a chance?

File: BiomarinPKU.studyApp/BiomarinPKU.studyApp/BiomarinPKU_studyApp.xcdatamodeld/.xccurrentversion [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>_XCCurrentVersionName</key>
+	<string>BiomarinPKU_studyApp.xcdatamodel</string>
```
Comment (syoung-smallwisdom) [inline BiomarinPKU.studyApp/BiomarinPKU.studyApp/BiomarinPKU_studyApp.xcdatamodeld/.xccurrentversion]: What do you need core data for?

File: BiomarinPKU.studyApp/BiomarinPKU.studyApp/Info.plist [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+	<array>
+		<string>armv7</string>
+	</array>
+	<key>UISupportedInterfaceOrientations</key>
+	<array>
+		<string>UIInterfaceOrientationPortrait</string>
+		<string>UIInterfaceOrientationLandscapeLeft</string>
+		<string>UIInterfaceOrientationLandscapeRight</string>
+	</array>
+	<key>UISupportedInterfaceOrientations~ipad</key>
```
Comment (syoung-smallwisdom) [inline BiomarinPKU.studyApp/BiomarinPKU.studyApp/Info.plist]: Should iPad be included?

File: BiomarinPKU.studyApp/BiomarinPKU.studyApp/BiomarinPKU_studyApp.xcdatamodeld/.xccurrentversion (hunk 1 above)
Comment (DanWebster) [reply BiomarinPKU.studyApp/BiomarinPKU.studyApp/BiomarinPKU_studyApp.xcdatamodeld/.xccurrentversion]: Don't need it, should I remove here?

File: BiomarinPKU.studyApp/BiomarinPKU.studyApp/Info.plist (hunk 2 above)
Comment (DanWebster) [reply BiomarinPKU.studyApp/BiomarinPKU.studyApp/Info.plist]: nope!

File: BiomarinPKU.studyApp/BiomarinPKU.studyApp/BiomarinPKU_studyApp.xcdatamodeld/.xccurrentversion (hunk 1 above)
Comment (syoung-smallwisdom) [reply BiomarinPKU.studyApp/BiomarinPKU.studyApp/BiomarinPKU_studyApp.xcdatamodeld/.xccurrentversion]: Yes, please

**Comment by DanWebster** [comment]
@syoung-smallwisdom can you add me, Erin, Michael DePhillips and Eric Sieg to have merge permissions on this repo?

**Comment by DanWebster** [comment]
Going to close this PR without merging and just re-create the project without the core data and universal app configurations checked during startup.

**Pull request closed without merging**